deepmd/pt/train/training.py

```
@@ -1260,15 +1281,24 @@ def get_single_model(
     return model
 
 
-def get_model_for_wrapper(_model_params):
+def get_model_for_wrapper(
+        _model_params,
+        _loss_params=None,
+):
     if "model_dict" not in _model_params:
+        if _loss_params is not None:
+            if whether_hessian(_loss_params):
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)